### DISCUSSION TITLE
Due to broken IRC communication right now

### DISCUSSION THREAD

**Discussion by Consolatis** [labwc/labwc#835] [Team Posts]
@johanmalm 

Would be great if you could clarify the project stance on "basic effects":
labwc/labwc/issues/805

Personally, I wouldn't mind merging a very simple non-intrusive default-disabled shadows PR (I wouldn't work on it myself though).
But I also wouldn't mind if the official project stance on that is a plain "no".

**Comment by jlindgren90** [comment]
Personally I'd prefer if we said no to effects (even shadows) at least until we've had more time to stabilize/shake out more of the bugs in our existing feature-set.

**Comment by johanmalm** [comment]
Okay. Will respond.
Agree - there is way too much to sort out to get to 'production code' without adding bling.
We've explicitly said no to animations and gradients in the 'README' anyway.

Looks like bnc4free-orange has been down for >7 days now, so maybe I need to connect via another bouncer.
https://status.bnc4free.com/
I've not really hit the ground for nearly two weeks anyway, so hasn't made a big difference :smile:
